server/client.go

```diff
@@ -286,7 +286,7 @@ func (client JiraClient) SearchCommentVisibilityFields(params map[string]string)
 	if err := client.RESTGet(commentVisibilityRoute, params, result); err != nil {
 		return nil, err
 	}
-	result.Groups.JiraUserGroups = append(result.Groups.JiraUserGroups, JiraUserGroup{visibleToAllUsers})
+	result.Groups.JiraUserGroups = append(result.Groups.JiraUserGroups, &JiraUserGroup{visibleToAllUsers})
```

Comment:
Have you tested the plugin after making these changes?

Reply:
Yeah, it is working perfectly fine.